ixofoundation/ixo-webclient
# feat(cells): listing page

## Scope

- https://nonaredteam.atlassian.net/browse/IXO-459

## Work Done

- Refactored entities again
- Added cell type with filters and relevant info to cells

## Steps to Test

- Run project
- Switch to cells
- See that everything changes to cells and that the filters are loaded with cell relevant data and filters are reset
- See that if you are logged in, on first load and when you reset filters, you will get My Portfolio and if you aren't you will get Featured by default

NOTE: Because the projects/cells do not currenty have an entityType set, they will not actually select cells or projects, otherwise we will get no projects or cells at all. Once they add some we can add a filter to the selectors to finish it off.
